docs/general/builtins/scalar-functions.rst

```diff
@@ -4278,6 +4278,31 @@ Example:
     SELECT 1 row in set (... sec)
 
 
+.. _scalar-pg_table_is_visible:
+
+``pg_table_is_visible()``
+-------------------------
+
+The function ``pg_table_is_visible`` returns ``true`` if the current user holds
+``DQL`` privilege on the table and there are no other tables with the same
```

Comment:
Have you double-checked with postgres?
What happens if the user has `DML` or `DDL` and not `DQL`, does it return false?

Reply:
> Have you double-checked with postgres?

Privileges seem to work differently for Postgres. `select` on table without `usage` on schema, one cannot select on the table(but CrateDB only enforces table level privilege to query tables). Similarly, for `pg_table_is_visible`, Postgres requires `usage` on schema but I thought for CrateDB we should just require table level privilege (DQL). We could also have a team discussion.

> What happens if the user has DML or DDL and not DQL, does it return false?

Yes, but now I see that it should return `true`. If one can execute any query on a table with unqualified name, it is equivalent to 'visible'.